.github/CONTRIBUTING.md

```diff
@@ -15,12 +15,45 @@ To contribute,
 
 ## Setting up the Local Development Environment
 
+### Get the source code
 1. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) this repository
 2. Open your favorite command line tool and navigate to the directory you wish to clone this repository to: `cd /path/to/clone`
 3. [Clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) your fork: `git clone git@github.com:{your username}/hospitalrun-frontend.git`
 4. Navigate to the hosptialrun-frontend directory: `cd hospitalrun-frontend`
-5. Install dependencies: `yarn`
-6. Run the application `yarn start`
+
+### Configure CouchDB
+CouchDB is the server side database which data from the frontend will sync to. In order to login
+to HospitalRun, CouchDB is required. For convienence, we have added a docker compose file in the
+root of this project to help launch CouchDB. However, you could install and run CouchDB in any way you wish.
```

Comment:
I might add:
DB created has no data/configuration persistence. Once the container is stopped, it is necessary to perform the initialization from step 3.

Reply:
I'm fairly certain that it has data persistence after I shut down the container. Maybe I'm mistaken?

Stopping the container does not clear the persistence, however doing a `docker rm` on the container will. 

I'm no docker expert though, so maybe I'm misunderstanding.

Reply:
Yeah, right. I made, without thinking about it,  a compose down for the cleanup